app/javascript/gobierto_dashboards/webapp/mixins/get_remote_data.js

```diff
@@ -0,0 +1,13 @@
+import * as d3 from 'd3'
```

Comment:
Look at the gobierto_data factories. This must be similar.

- Don't fetch `d3`. Use axios instead
- Declare here the endpoints

Reply:
Also move the file inside a `lib` folder to distinguish from other stuff